### PR TITLE
Add BandLab blog post

### PR DESCRIPTION
**Pull request by nicka**
@Stefmonge It's still pending company review! I'll let you know when it may be published. Btw I randomly selected the date, feel free to adjust :+1:

### CODE REVIEW

**Comment by Stefmonge** [comment]
Sounds good @nicka! Thanks for the update!

**Comment by nicka** [comment]
@Stefmonge It has been reviewed and is ready to be published 🎉

**Comment by Stefmonge** [comment]
Thanks @nicka! I'll have a look and get back to you with a publish date in the next day or so.

**Comment by nicka** [comment]
Thanks for the updates 👍

**Comment by nicka** [comment]
@Stefmonge any news on this?

**Comment by Stefmonge** [comment]
Hey @nicka! It's scheduled for next Thursday, Nov. 24. I'll make sure you see any additional edits before we publish.

**Comment by Stefmonge** [comment]
@nicka Ready to go once you approve!

File: posts/2016-11-10-going-serverless-at-bandlab.md [hunk 1]
```diff
@@ -42,34 +46,36 @@ While still at the Dutch company we managed to set up a solid framework to autom
...
 
-Internally we started sprinkling in custom [Lambda-Backed Custom Resources](https://aws.amazon.com/blogs/aws/aws-cloudformation-update-lambda-backed-custom-resources-more/) within CloudFormation stacks to fill-up missing gaps. This was when I started to realise how powerful Lambda actually is. We were able to automate a lot of use-cases and 3rd party applications/providers with the use of AWS Lambda.
+Internally we started sprinkling in custom [Lambda-Backed Custom Resources](https://aws.amazon.com/blogs/aws/aws-cloudformation-update-lambda-backed-custom-resources-more/) within CloudFormation stacks to fill-up missing gaps. This was when I started to realize how powerful Lambda actually is. We were able to automate a lot of use-cases and 3rd party applications/providers with the use of AWS Lambda.
+
+![pexels-photo-38125](https://cloud.githubusercontent.com/assets/20538501/20605146/70cadfb6-b22f-11e6-8252-80593cfef88d.jpeg)
```
Comment (nicka) [inline posts/2016-11-10-going-serverless-at-bandlab.md]: Please remove the image at this section, as this still refers about the previous company I worked at(has nothing to do with audio). If you would like to add an image somewhere, please use this one https://blog.bandlab.com/wp-content/uploads/2016/11/new_33b.jpg which comes directly from our BandLab blog.

File: posts/2016-11-10-going-serverless-at-bandlab.md [hunk 2]
```diff
@@ -42,34 +46,36 @@ While still at the Dutch company we managed to set up a solid framework to autom
...
+>As [AWS API Gateway](https://aws.amazon.com/api-gateway/) and AWS Lambda were maturing, I decided to invest my time in using these newer styles of computing, and prove that it is entirely possible to design complete applications without having to manage any servers.
 
-The Serverless Framework seemed like the perfect fit to accomplish this goal! Noticing that they used CloudFormation from the start also gave me the assurance that deployments would be straight-forward and battle tested.
+The Serverless Framework seemed like the perfect fit to accomplish this goal! Noticing that it used CloudFormation from the start also gave me the assurance that deployments would be straight-forward and battle tested.
 
 **Today**
 
 >A creative, collaborative, global community of Musicians
 
-[BandLab](https://www.bandlab.com) is where musicians and music fans from all over the world can come together to make, collaborate and share Music. The platform works on both iOS and Android. Everything users create is hosted in the cloud, so they can capture musical ideas wherever they are, then further develop them using BandLab’s web-based, MIDI-enabled app.
```
Comment (nicka) [inline posts/2016-11-10-going-serverless-at-bandlab.md]: May I ask why "The platform works on both iOS and Android." was removed?

File: posts/2016-11-10-going-serverless-at-bandlab.md (hunk 1 above)
Comment (nicka) [inline posts/2016-11-10-going-serverless-at-bandlab.md]: @Stefmonge Please review

File: posts/2016-11-10-going-serverless-at-bandlab.md (hunk 2 above)
Comment (nicka) [inline posts/2016-11-10-going-serverless-at-bandlab.md]: @Stefmonge Please review

**Comment by nicka** [comment]
@Stefmonge Did you notice my comments?

>Please remove the image at this section, as this still refers about the previous company I worked at(has nothing to do with audio). If you would like to add an image somewhere, please use this one https://blog.bandlab.com/wp-content/uploads/2016/11/new_33b.jpg which comes directly from our BandLab blog.

>May I ask why "The platform works on both iOS and Android." was removed?

**Comment by Stefmonge** [comment]
@nicka Making those changes right now.

**Comment by nicka** [comment]
@Stefmonge Thanks, once updated I'm good to go 👍

**Pull request merged**